.rubocop.yml

```diff
@@ -42,6 +42,8 @@ Rails/FilePath:
   Enabled: false
 Style/Documentation:
   Enabled: false
+Style/FormatString:
```

Comment:
We have format as an ovverride for formatted descriptions/expressions. Fixing these violations would break custom formatting